### ISSUE TITLE
i cant find the stl files for CQ-80

### ISSUE THREAD

**Issue by JVGWTX** [casouri/cq-80#1]
i searched everywhere and couldent find it does someone know where  i can find the files for the cq-80

**Comment by casouri** [comment]
You can find them here: https://github.com/casouri/cq-80/releases

**Comment by JVGWTX** [comment]
Thank you so much : )

**Comment by casouri** [comment]
Keep me updated! I'm curious of what you can make! Free free to ask me questions, too.

**Comment by JVGWTX** [comment]
@casouri ![image](https://github.com/casouri/cq-80/assets/147232740/68d8f9a3-8a62-4e8f-bd74-039441e72833)![Uploading 278120659...] well I'm in the process of building the CQ-80 and I wanted to thank you for the files and for your help it already looks beautiful but I'm going to make not the model of agent 8 I am going to make the model of Agent 3
In Octo Expansion thank you.

**Comment by casouri** [comment]
That'll be very interesting to see! So it's gonna be blue, huh?

**Comment by JVGWTX** [comment]
Yes exactly it's going to be blue but I need to reprint the secondary parts because they came not very good out of my printer but right now I'm in the process of sanding it down.
